system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,13 +321,54 @@
 }
 
 .topsite-form {
+  .form-input-container {
+    display: flex;
+    max-width: 492px;
+    margin: 0 auto;
+    padding: 32px;
+
+    .top-site-outer {
+      width: 96px;
+      padding: 0;
+      margin: 64px 0 0;
+      margin-inline-start: 32px;
+      list-style: none;
```

Comment:
Seems odd that we need to specially do this here for the form. Looks like the list normally avoids this by `display: inline-block;`.. maybe it should just happen for all `.top-site-outer`?

Reply:
Updated so that all `.top-site-outer` have display inline.